shipyard_blueprints/hightouch/pyproject.toml

```diff
@@ -9,7 +9,7 @@ packages = [{include = "shipyard_hightouch"}]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-requests = "2.28.0"
```

Comment:
Add `shipyard-templates` to both the dev dependency group (do it as a path in that case i.e. `../../shipyard-templates`) and to the main dependency group. You may also need to add shipyard-utils to the main dependency group